lapack/testlapack/dlange.go

```diff
@@ -34,16 +34,22 @@ func DlangeTest(t *testing.T, impl Dlanger) {
 		if lda == 0 {
 			lda = n
 		}
+		// Allocate m×n matrix A and fill it with random numbers from [-0.5, 0.5).
 		a := make([]float64, m*lda)
 		for i := range a {
 			a[i] = rnd.Float64() - 0.5
 		}
+		aCopy := make([]float64, len(a))
```

Comment:
In other cases where we have this, you've noted that this is for later comparison.

Reply:
I started doing only in the later commits. Added.